### PR TITLE
[FIX] modules_odoo.py: The way to skip the xml check is detecting the method called

### PR DESCRIPTION
**Pull request by JesusZapata**
This change is neccesary because the funcionality to skip the xml check detect the method than called the another method `_skip_files_ext`

The way I found to know the method than called was the package [inspect](https://docs.python.org/2/library/inspect.html)

https://github.com/Vauxoo/pylint-odoo/blob/master/pylint_odoo/misc.py#L273-L277

### CODE REVIEW

File: pylint_odoo/checkers/modules_odoo.py [hunk 1]
```diff
@@ -823,17 +823,14 @@ def _get_manifest_referenced_files(self):
                 referenced_files[fname] = data_type
         return referenced_files
 
-    def _get_module_files(self):
+    def _check_file_not_used(self):
+        """Check if a file is not used from manifest"""
+        self.msg_args = []
```
Comment (moylop260) [inline pylint_odoo/checkers/modules_odoo.py]: Use tuple and use `self. module_files.add()` instead of `self. module_files.append()`

**Comment by moylop260** [comment]
I don't understand how this is working and before not.
Could you explain me, please?

**Comment by JesusZapata** [comment]
@moylop260 
In the method `_skip_files_ext` is where the XML is reader and detect the magic coment `<!-- pylint-odoo:disable=xml-pylint-odoo-check -->`
https://github.com/Vauxoo/pylint-odoo/blob/cb7cc012baff15a975c1df931b23761f782572cd/pylint_odoo/misc.py#L267-L270

The way I found to detect what is the name of the check was using the `inspect` 
The `inspect` say which is the stack call and with this stack call I calculated the name of the check 

https://github.com/Vauxoo/pylint-odoo/blob/cb7cc012baff15a975c1df931b23761f782572cd/pylint_odoo/misc.py#L273-L276

I change this PR to better way to calculate the name of the check, I think of that way is less invasive to the existing code

File: pylint_odoo/misc.py [hunk 2]
```diff
@@ -271,7 +271,8 @@ def _skip_files_ext(self, fext, fnames):
         if fext != '.xml':
             return fnames
         info_called = [item[3] for item in inspect.stack() if
-                       'modules_odoo' in item[1]]
+                       'modules_odoo' in item[1]
+                       and item[3].startswith('_check_')]
```
Comment (JesusZapata) [inline pylint_odoo/misc.py]: @moylop260 
Now I find in the stack call only the method start with `_check_` the method named `_check_` are the method related with one check

Comment (moylop260) [reply pylint_odoo/misc.py]: and the tests that you had in the other commit?
In fact, you shouldn't squash commit of a PR in revision :(

**Comment by moylop260** [comment]
OCA PR please

**Pull request merged**